ci/pipeline.yml

```diff
@@ -115,7 +116,7 @@ jobs:
   - in_parallel:
     - get: broker-src
       passed: [acceptance-tests-staging]
-      trigger: true
+      # trigger: true
```

Comment:
This needs to be uncommented.

Reply:
Given the test coverage is fairly light, I believe leaving this commented is safer.

Reply:
Talked with Mike and reverted this.